src/components/implementation/Makefile.subsubdir

```diff
@@ -80,7 +82,11 @@ $(COMPONENT): $(OBJS)
 %.o:%.c
 #	$(info |     [CC]   $<: Compiling)
 	@$(CC) $(INCLUDE) $(CFLAGS) -o $@ -c $<
-
+       
+%.o:%.cc
+#	$(info |     [CC]   $<: Compiling)
```

Comment:
It might be nice to note in the message that it's C++ compiling here. I'm not sure that's really necessary though.